cmake/BLTMacros.cmake

```diff
@@ -664,6 +665,12 @@ macro(blt_add_test)
              COMMAND        ${test_command}
              CONFIGURATIONS ${arg_CONFIGURATIONS})
 
+    # Handle OpenMP
+    if( arg_NUM_OMP_THREADS )
+        set_property(TEST ${arg_NAME}
+                     APPEND PROPERTY ENVIRONMENT OMP_NUM_THREADS=${arg_NUM_OMP_THREADS})
```

Comment:
Really nice!

Presumably, this would override an existing value in the user's environment.

Does this property only apply when a user runs ``ctest`` or ``make test``?

Reply:
It is my understanding that this only affects the named test.